tools/rostest/nodes/paramtest

```diff
@@ -90,7 +90,7 @@ class ParamTest(unittest.TestCase):
 
         wallclock_timeout_t = time.time() + wait_time
         param_obtained = None
-        while not param_obtained and time.time() < wallclock_timeout_t:
+        while param_obtained in None and time.time() < wallclock_timeout_t:
```

Comment:
Please use `is None` here. As the failing tests show the current patch doesn't work because `None` isn't iterable. Please also make sure to run the modified code locally and confirm that it behaves as desired before creating a pull request.

Reply:
I am sorry, my bad. I have amended my original commit with your suggestions after running the tests locally.